src/main/java/seedu/address/logic/parser/TagCommandParser.java

```diff
@@ -19,18 +19,30 @@ public class TagCommandParser implements Parser<TagCommand> {
     public TagCommand parse(String args) throws ParseException {
         requireNonNull(args);
         ArgumentMultimap multimap = ArgumentTokenizer.tokenize(args, PREFIX_TAG);
+
+        //Reformatting error message for duplicate tags
         try {
-            Index index = ParserUtil.parseIndex(multimap.getPreamble());
-            String tags = multimap.getValue(PREFIX_TAG).orElse("");
-            Set<Tag> tagSet = Tag.stringToTagSet(tags);
+            multimap.verifyNoDuplicatePrefixesFor(PREFIX_TAG);
+        } catch (ParseException pe) {
+            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT,
+                    "Please only include one prefix t/ !"));
```

Comment:
Maybe could make this error message a constant MESSAGE_DUPLICATE_PREFIXES in TagCommand? Same for "No tags specified"